terragrunt/README.md

```diff
@@ -115,8 +115,37 @@ instance of the service will create its own certificate.
 
 ## Running Terragrunt
 
+When running [Terragrunt], it fetches and stores the state of the infrastructure
+in AWS. This requires that an AWS profile is configured for each account. After
+that has been done, running [Terragrunt] is as simple as replacing `terraform`
+with `terragrunt` in the command.
+
+### Configure AWS Profiles
+
 Running [Terragrunt] requires permissions to the AWS account you are
-configuring. Assuming you have permission, you can `cd` into the corresponding
+configuring. For each of the accounts, copy the following snippet into the
+configuration file at `~/.aws/config`:
+
+```text
+[profile rust-root]
+sso_start_url = https://rust-lang.awsapps.com/start
+sso_account_id = <rust-root-account-id>
+sso_role_name = AdministratorAccess
+sso_region = us-east-1
+region = us-east-1
+```
+
+Each account in `./accounts` has a `account.json` file that contains the name of
```

Comment:
This is only true of the root account I believe. Should we add account.json files for the other accounts?

Reply:
We've added that file to all accounts in https://github.com/rust-lang/simpleinfra/pull/168.